### ISSUE TITLE
Can only require a single 'active_*' gem in version 2.5.0

### ISSUE THREAD

**Issue by Fjan** [rubygems/rubygems#1405]
With Rubygems 2.5.0 the first require works normally, the second one fails for any gems from the Rails family:
```
irb> require 'active_record'  # or action_mailer, etc.
true
irb> require 'active_job'  # or active_support, etc.
LoadError: cannot load such file -- active_job
```

Subsequent ```require``` always fails for Railsie gems, but work fine for other gems. So perhaps its a Rails issue.

Downgrading to Rubygems 2.4.8 fixes the problem.

**Comment by segiddins** [comment]
Can you try master? I believe I've fixed this particular bug. If so, it might be worth getting out a patch release @drbrain ?

**Comment by Fjan** [comment]
Yes, this is fixed on 2.5.1 :+1:
